Pneumothorax losing breath sound

MissHeda opened the pull request:
Adds a sound that plays every 30s if you have any type of pneumothorax simulating you losing your breath to it.

MissHeda committed:
Pneumothorax losing breath sound

Adds a sound that plays every 30s if you have any type of pneumothorax simulating you losing your breath to it.

AtrixZockt commented on addons/breathing/functions/fnc_handleBreathing.sqf:
```diff
@@ -202,3 +202,29 @@ if (!local _unit) then {
+        [_idPFH] call CBA_fnc_removePerFrameHandler;
+    };
+
+    if (_unit getVariable [QGVAR(pneumothorax), false]) then {
```
If they all share the same sound put them into 1 if statement

MissHeda committed:
requested changes :3

AtrixZockt commented on addons/breathing/functions/fnc_handleBreathing.sqf:
```diff
@@ -202,3 +202,21 @@ if (!local _unit) then {
+
+    private _status = _unit getVariable [QGVAR(airwayStatus), 100];
+
+    if(!(alive _unit) || _status > 100 || _unit getVariable ["ACE_isUnconscious", false]) exitwith {
```
Use macros.

Instead of "_unit getVariable ["ACE_isUnconscious", false]"
use IS_UNCONSCIOUS(_unit)

MissHeda committed:
requested changes

MiszczuZPolski commented on addons/breathing/functions/fnc_handleBreathing.sqf:
```diff
@@ -202,3 +202,21 @@ if (!local _unit) then {
+
+    private _status = _unit getVariable [QGVAR(airwayStatus), 100];
+
+    if(!(alive _unit) || _status > 100 || IS_UNCONSCIOUS(_unit)) exitwith {
```
shouldnt it be "status >= 100"? i dont see a reason to keep that pfh running when someone SpO2 status is 100
and add space after if

MissHeda replied:
Thanks for the hint completly oversaw that.

MiszczuZPolski replied:
make it >= instead of == from last commit

MissHeda replied:
But SpO2 can not be above 100, correct?

MissHeda committed:
SpO2 FrameHandler remover update

MissHeda committed:
pneumothorax sound code cleanup

Everything should work, tested it ingame but please give it another try.

AtrixZockt reviewed (approved):
Haven't found anything that would need to be changed. But would be good if someone else can look over it too

MissHeda committed:
Calling breathing fix

Handle breathing was not called again after respawn, fixed that. Now everything is working.

Tomcat-SG reviewed (approved):
LGTM
